### PR TITLE
Add workflow to trigger CoC violation analysis

### PR DESCRIPTION
**Pull request by nitin-ramchandani**
Adding workflow to trigger CoC analysis, post `Lint` workflow is successful.

CoC analysis validates the contribution yml files against the following areas:

- Privacy (Personal Identifiable Information)
- HAP (Hate, Abuse, Profanity)
- Inappropriate Content (Adult, Illegal, Gambling)
- Regulated Industries (Medical, Financial)

### CODE REVIEW

File: .github/workflows/coc_violation_analysis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    name: Run annotator insights
+    steps:
+      - name: Invoke insight handler
+        id: run_insights
+        run: |
+          pull_request_number=$(jq --raw-output '.workflow_run.pull_requests[0].number' "$GITHUB_EVENT_PATH")
+          github_org=$(echo ${{ github.repository }} | cut -d "/" -f 1)
+          repo_name=$(echo ${{ github.repository }} | cut -d "/" -f 2)
```
Comment (bjhargrave) [inline .github/workflows/coc_violation_analysis.yml:19]:
```suggestion
```

File: .github/workflows/coc_violation_analysis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: CoC violation analysis
+run-name: Running CoC violation analysis
+on:
+  workflow_run:
+    workflows: [Lint]
```
Comment (bjhargrave) [inline .github/workflows/coc_violation_analysis.yml:5]:
```suggestion
    workflows:
      - Lint
```

**Review by bjhargrave — CHANGES_REQUESTED** [review]

**Comment by bjhargrave** [comment]
I guess this got merged before my review was complete. I'll make a new PR.

**Comment by jjasghar** [comment]
Yep, we wanted to test this ASAP, so I went ahead and merged it. I had to put the SECRET in and all and the only way was to get it merged.

**Pull request merged**